wiki/Download.md

```diff
@@ -112,7 +111,7 @@ list](Mailing_lists "wikilink") and ask for help there.
 
 ### Required Software
 
--   [Python 2.7, 3.4, 3.5, 3.6, or 3.7](http://www.python.org) or PyPy,
+-   [Python 3.6, 3.7 or higher](http://www.python.org) or PyPy,
```

Comment:
Either just "3.6 or later" (although not future proof) or "3.6, 3.7 or 3.8" to be explicit.